cpp/oneapi/dal/backend/primitives/placement/argwhere_dpc.cpp

```diff
@@ -0,0 +1,242 @@
+/*******************************************************************************
+* Copyright 2023 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "oneapi/dal/detail/profiler.hpp"
+
+#include "oneapi/dal/backend/primitives/common.hpp"
+#include "oneapi/dal/backend/primitives/ndarray.hpp"
+#include "oneapi/dal/backend/primitives/reduction.hpp"
+
+#include "oneapi/dal/backend/primitives/placement/argwhere.hpp"
+
+namespace oneapi::dal::backend::primitives {
+
+template <typename Binary, typename Type, typename Index, where_alignment align>
+sycl::event argextreme(sycl::queue& queue,
```

Comment:
Please add a description.

Reply:
It is miscellaneous functionality hidden in the `.cpp` file. It's developer-facing counterparts are all documented and I see no point in explaining the same thing twice and doing it in hidden functionality